specification/2.0/Specification.adoc

```diff
@@ -1164,31 +1164,28 @@ The following example defines a mesh containing one indexed triangle set primiti
 
 Each attribute is defined as a property of the `attributes` object. The name of the property corresponds to an enumerated value identifying the vertex attribute, such as `POSITION`. The value of the property is the index of an accessor that contains the data.
 
-The specification defines the following attribute semantics: `POSITION`, `NORMAL`, `TANGENT`, `TEXCOORD_0`, `TEXCOORD_1`, `COLOR_0`, `JOINTS_0`, and `WEIGHTS_0`.
+The specification defines the following attribute semantics: `POSITION`, `NORMAL`, `TANGENT`, `TEXCOORD_n`, `COLOR_n`, `JOINTS_n`, and `WEIGHTS_n`.
 
 Application-specific attribute semantics **MUST** start with an underscore, e.g., `_TEMPERATURE`. Application-specific attribute semantics **MUST NOT** use __unsigned int__ component type.
 
 Valid accessor type and component type for each attribute semantic property are defined below.
 
 [options="header",cols="15%,20%,35%,30%"]
 |====
-| Name        | Accessor Type(s)| Component Type(s)| Description
-| `POSITION`  | `"VEC3"`        | _float_                     | XYZ vertex positions
-| `NORMAL`    | `"VEC3"`        | _float_                     | Normalized XYZ vertex normals
-| `TANGENT`   | `"VEC4"`        | _float_                     | XYZW vertex tangents where the W component is a sign value (-1 or +1) indicating handedness of the tangent basis
-| `TEXCOORD_0`| `"VEC2"`        | _float_ +
+| Name        | Accessor Type(s)| Component Type(s)           | Description
+| `POSITION`  | VEC3            | _float_                     | XYZ vertex positions
+| `NORMAL`    | VEC3            | _float_                     | Normalized XYZ vertex normals
+| `TANGENT`   | VEC4            | _float_                     | XYZW vertex tangents where the W component is a sign value (-1 or +1) indicating handedness of the tangent basis
+| `TEXCOORD_n`| VEC2            | _float_ +
                                   _unsigned byte_ normalized +
-                                  _unsigned short_ normalized | UV texture coordinates for the first set
-| `TEXCOORD_1`| `"VEC2"`        | _float_ +
-                                  _unsigned byte_ normalized +
-                                  _unsigned short_ normalized | UV texture coordinates for the second set
-| `COLOR_0`   | `"VEC3"` +
-                `"VEC4"`        | _float_ +
+                                  _unsigned short_ normalized | UV texture coordinates
+| `COLOR_n`   | VEC3 +
+                VEC4            | _float_ +
                                   _unsigned byte_ normalized +
                                   _unsigned short_ normalized | RGB or RGBA vertex color linear multiplier
-| `JOINTS_0`  | `"VEC4"`        | _unsigned byte_
+| `JOINTS_n`  | VEC4            | _unsigned byte_
```

Comment:
`JOINTS_n` are integers, IDs of particular bones in the rig.  You can't morph from bone 3 to bone 5; who knows if bone 4 is anything related to 3 or 5.

Reply:
This table describes the base mesh accessors, not morph targets.